### PR TITLE
Fix: Background gradient color dynamic tag is not being rendered on the frontend [ED-15218]

### PR DESCRIPTION
**Pull request by louiswol94**

### CODE REVIEW

File: core/files/css/base.php [hunk 1]
```diff
@@ -780,7 +787,9 @@ private function get_style_control_value( array $control, array $values ) {
 			return $this->get_selector_global_value( $control, $values['__globals__'][ $control['name'] ] );
 		}
 
-		$value = $values[ $control['name'] ];
+		$value = isset( $values[ $control['name'] ] )
+			? $values[ $control['name'] ]
+			: $this->preserved_dynamic_style_values[ $control['name'] ];
```
Comment (KingYes) [inline core/files/css/base.php:792]: @louiswol94 You should check isset on `$this->preserved_dynamic_style_values` too. No?

**Review by KingYes — CHANGES_REQUESTED** [review]

File: core/files/css/base.php (hunk 1 above)
Comment (KingYes) [reply core/files/css/base.php:792]: You're right, but still, how you can make sure the value is set there?

Comment (louiswol94) [reply core/files/css/base.php:792]: @KingYes Let me add null as a fallback? :)

Comment (KingYes) [reply core/files/css/base.php:792]: I think it's okay. Thanks.

Comment (louiswol94) [reply core/files/css/base.php]: @KingYes Updated.

**Review by KingYes — APPROVED** [review]

**Pull request merged**